core/src/main/java/io/aiven/klaw/helpers/db/rdbms/UpdateDataJdbc.java

```diff
@@ -198,16 +198,15 @@ public String updateConnectorRequest(KafkaConnectorRequest connectorRequest, Str
     topicObj.setHistory(connectorRequest.getHistory());
     connectors.add(topicObj);
 
-    if (connectorRequest.getRequestOperationType().equals(RequestOperationType.CREATE.value)) {
-      insertDataJdbcHelper.insertIntoConnectorSOT(connectors, false);
-    } else if (connectorRequest
-        .getRequestOperationType()
-        .equals(RequestOperationType.UPDATE.value)) {
-      updateConnectorSOT(connectors, connectorRequest.getOtherParams());
-    } else if (connectorRequest
-        .getRequestOperationType()
-        .equals(RequestOperationType.DELETE.value)) {
-      deleteDataJdbcHelper.deleteConnectors(topicObj);
+    final RequestOperationType rot =
+        RequestOperationType.of(connectorRequest.getRequestOperationType());
+    if (rot == null) {
+      return ApiResultStatus.SUCCESS.value;
+    }
+    switch (rot) {
+      case CREATE -> insertDataJdbcHelper.insertIntoConnectorSOT(connectors, false);
```

Comment:
I'm just wondering about if Promote is missing here as it was in #730 going to take a quick look and see if I can figure it out.

Reply:
As you can see in the screenshot below the connectortype has not yet been updated to Promote.
We should raise a ticket to update both here and there with the correct RequestOperationType.
![image](https://user-images.githubusercontent.com/121855584/222362326-ba9af4f1-dad7-4891-869c-578505ccbc3e.png)